app/bundles/FormBundle/Views/Field/field_helper.php

```diff
@@ -161,6 +161,16 @@
         }
     }
 
+    if ($field['leadField'] && in_array($contactFields[$field['leadField']]['type'], ['datetime', 'date'])) {
```

Comment:
I would sleep better if this condition was:
```
if (!empty($field['leadField']) && !empty($contactFields[$field['leadField']]['type']) && in_array($contactFields[$field['leadField']]['type'], ['datetime', 'date'])) {
```
I wasn't able to break it, but there might exist an use case where the array properties doesn't exist.